### ISSUE TITLE
NAs in the 2019 maps

### ISSUE THREAD

**Issue by danmackinlay** [ropenscilabs/eechidna#34]
There are some weird NAs in the 2019 maps that are not in the 2016 map. Are these a problem?

``` r
library(dplyr)
library(eechidna)
nat_map16 = nat_map_download(2016)
glimpse(nat_map16)
#> Rows: 37,041
#> Columns: 9
#> $ long      <dbl> 130.5179, 130.5175, 130.5294, 130.5243, 130.5310, 130.5252,…
#> $ lat       <dbl> -12.73105, -12.72562, -12.72505, -12.73409, -12.74217, -12.…
#> $ order     <int> 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, …
#> $ hole      <lgl> FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FAL…
#> $ piece     <chr> "P.1", "P.1", "P.1", "P.1", "P.1", "P.1", "P.1", "P.1", "P.…
#> $ id        <chr> "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1",…
#> $ group     <chr> "G.1.1", "G.1.1", "G.1.1", "G.1.1", "G.1.1", "G.1.1", "G.1.…
#> $ elect_div <chr> "LINGIARI", "LINGIARI", "LINGIARI", "LINGIARI", "LINGIARI",…
#> $ state     <chr> "NT", "NT", "NT", "NT", "NT", "NT", "NT", "NT", "NT", "NT",…
sum(is.na(nat_map16$elect_div))
#> [1] 0
nat_map19 = nat_map_download(2019)
glimpse(nat_map19)
#> Rows: 33,427
#> Columns: 9
#> $ long      <dbl> 149.2063, 149.1382, 149.1551, 149.1355, 149.1515, 149.1314,…
#> $ lat       <dbl> -35.34591, -35.42940, -35.43578, -35.45442, -35.49224, -35.…
#> $ order     <int> 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, …
#> $ hole      <lgl> FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FAL…
#> $ piece     <chr> "P.1", "P.1", "P.1", "P.1", "P.1", "P.1", "P.1", "P.1", "P.…
#> $ id        <chr> "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0",…
#> $ group     <chr> "G.0.1", "G.0.1", "G.0.1", "G.0.1", "G.0.1", "G.0.1", "G.0.…
#> $ elect_div <chr> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA,…
#> $ state     <chr> NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA, NA,…
sum(is.na(nat_map19$elect_div))
#> [1] 92
```

Note that there are some geographic objects whose `elect_div` and `state` columns are NA in 2019 only.
This is different to the 2016 version, which seems to always have content there.
I'm not yet geo-savvy enough to know exactly how to parse these weird geographic data frames, but if I work this out while playing with the data I'll post more information.

**Comment by jforbes14** [comment]
Looks like this might have just been an error missed when the 2019 map object was generated. I've regenerated it in my `hotfix-refactor-wrangling` branch, which now doesn't have any missing values. I'll merge with `master` soon (just waiting on another PR first), and not long after we should have a new release on CRAN.

I'll keep this issue open until it's in master

**Comment by jforbes14** [comment]
Fixed in `master`, closing issue
